src/styles/index.scss

```diff
@@ -1,3 +1,224 @@
+@import './variables';
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.header {
```

Comment:
create separate files for all of BEM blocks (header, nav, container, card, stars)